workflows/methylation/methylation-cohort.wdl

```diff
@@ -0,0 +1,251 @@
+version 1.1
+
+workflow methylation_cohort {
+    meta {
+        name: "Methylation Cohort"
+        description: "Process methylation data for a cohort of samples"
+        outputs: {
+            combined_beta: "Matrix (in CSV format) containing beta values for every (common) probe on the array as rows and all of the input samples as columns.",
+            filtered_beta: "Matrix (in CSV format) containing only beta values for the retained probes (top N highest standard deviation) for all provided samples.",
+            filtered_probeset: "List of probe names that were retained after filtering to the top N highest standard deviation",
+            umap_embedding: "UMAP embedding for all samples",
+            umap_plot: "UMAP plot for all samples",
+        }
+    }
+
+    parameter_meta {
+        unfiltered_normalized_beta: "Array of unfiltered normalized beta values for each sample"
+    }
+
+    input {
```

Comment:
Should `num_probes` be exposed at the workflow level? It seems reasonable to want to tweak that value.